arangograph/single-sign-on.md

```diff
@@ -9,9 +9,28 @@ title: Single Sign-On (SSO)
 This feature is only available in ArangoGraph Enterprise.
 {% endhint %}
 
-ArangoGraph provides support to enable Single Sign-On (SSO) authentication using
-SAML 2.0 with SCIM provisioning. This enables you to propagate to ArangoGraph any
-user access changes by using the dedicated API.
+ArangoGraph provides support to enable **Single Sign-On** (SSO) authentication
+using **Security Assertion Markup language 2.0** (SAML 2.0) with the
+**System for Cross-domain Identity Management** (SCIM) provisioning. 
+This enables you to propagate to ArangoGraph any user access changes by using
+the dedicated API.
+
+{% hint 'info' %}
+To enable the Single Sign-On (SSO) feature, please
+[contact us](https://www.arangodb.com/contact){:target="_blank"}.
```

Comment:
Can we change this to pointing them towards their ArangoDB account team, or to an ArangoGraph Support ticket please?

Reply:
Yes, let's instruct them to always share these via an ArangoGraph Support ticket.